flipperdevices/flipperzero-firmware
# 1342 add mifare infineon

# What's new
As per Issue #1342, this PR adds support for MIFARE Classic 1K chipsets from the INFINEON manufacturer.
NXP's SAK is 0x08, INFINEON's is 0x88

# Verification 

- Compile & Flash
- In the NFC Menu, "Read Card"
- Place a Mifare Classic 1K card with an SAK value of 0x88
- Confirm the card is detected as a Mifare Classic 
- Confirm that the 'Run Compatible App" is displayed on the next screen

![image](https://user-images.githubusercontent.com/2514540/176004147-9b21fa47-1651-4d43-89bd-54003b6e2f56.png)
![image](https://user-images.githubusercontent.com/2514540/176004380-673ff95d-2cb4-411f-bf3c-4a0fa1e8ce77.png)

- Confirm that the emulated badge emulates 0x88 as SAK:
```
nfc-mfclassic r a u flip.dmp infineon.dmp
NFC reader: NXP / PN533 opened
Found MIFARE Classic card:
ISO/IEC 14443A (106 kbps) target:
    ATQA (SENS_RES): 00  04
       UID (NFCID1): 84  85  de  ab
      SAK (SEL_RES): 88
```
# Checklist (For Reviewer)
